example/src/Pagination.js

```diff
@@ -0,0 +1,115 @@
+import { getPagination } from '@kaliber/pagination'
+import styles from './Pagination.css'
+
+export function Pagination({ current, max, deriveUrl, onChange }) {
+
+  // Semantics: https://www.a11ymatters.com/pattern/pagination/
+  return (
+    <nav className={styles.component} aria-label='Pagination'>
+      <Arrow
+        page={current + 1}
+        onClick={() => handleClick(current + 1)}
+        layoutClassName={styles.next}
+        label={`Next page (page ${current + 1})`}
+        dataX='goto-next-page'
+        {...{ max, deriveUrl }}
+      >
+        &rarr;
+      </Arrow>
+
+      <Arrow
+        page={current - 1}
+        onClick={() => handleClick(current - 1)}
+        layoutClassName={styles.previous}
+        label={`Previous page (page ${current - 1})`}
+        dataX='goto-previous-page'
+        {...{ max, deriveUrl }}
+      >
+        &larr;
+      </Arrow>
+
+      <Bullets onBulletClick={handleClick} {...{ current, max, deriveUrl }} />
+    </nav>
+  )
+
+  function handleClick(x) {
+    if (x !== current) onChange(x)
+  }
+}
+
+function Bullets({ current, max, deriveUrl, onBulletClick }) {
+  const pagination = getPagination({ current, max, padding: 1 })
+
+  return (
+    <ul className={styles.componentBullets}>
+      {pagination.map((x, i) => typeof x === 'number' ? (
+        <li key={i}>
+          <Bullet
+            page={x}
+            url={deriveUrl(x)}
+            active={current === x}
+            onClick={() => onBulletClick(x)}
+            dataX={`goto-page-${x}`}
+          />
+        </li>
+      ) : (
+        <li role="presentation" className={styles.gap}>&hellip;</li>
+      ))}
```

Comment:
Ik dacht dat onze linting iets afdwong over `?:`, en dat we deze keuzes hadden:

```
// 1
a ? b : c

// 2
a
  ? b
  : c

// 3
a ? b :
c ? d :
e 
```

Reply:
Dit mag volgens onze linting ook, zo wordt het vaak gedaan omdat dit is hoe prettier het format. Maar ik zal m aanpassen naar manier 2!